src/modules/form/components/FormDialogActionContent.tsx

```diff
@@ -1,6 +1,19 @@
 import { LoadingButton } from '@mui/lab';
 import { Box, Button, ButtonProps, Stack } from '@mui/material';
-import { ReactNode } from 'react';
+import React, { ReactNode, useMemo } from 'react';
+import ButtonTooltipContainer from '@/components/elements/ButtonTooltipContainer';
+
+export interface FormDialogActionProps {
+  onDiscard: ButtonProps['onClick'];
+  onSubmit: ButtonProps['onClick'];
+  submitLoading?: boolean;
+  discardButtonText?: string;
+  submitButtonText?: string;
+  disabled?: boolean;
+  disabledReason?: string;
+  otherActions?: ReactNode;
+  PrimaryActionProps?: ButtonProps;
```

Comment:
Maybe `SubmitButtonProps`? Afaik we don't much refer to the submit action as "primary". I imagine it might be confusing with "primary" also being used in theming.